Backpack-SwiftUI/Tests/SnapshotHelpers/SnapshotHelpers.swift

```diff
@@ -30,9 +30,16 @@ func assertSnapshot<V: View>(
 ) {
     let identifier: (UIUserInterfaceStyle) -> String = { mode in mode == .dark ? "dark-mode" : "light-mode" }
     modes.forEach { mode in
-        let vc = UIHostingController(rootView: view)
+        let view: UIView = UIHostingController(rootView: view).view
         let styleTrait = UITraitCollection(userInterfaceStyle: mode)
         isRecording = false
-        assertSnapshot(matching: vc, as: .image(traits: styleTrait), named:identifier(mode), file: file, testName: testName, line:line)
+        assertSnapshot(
+            matching: view,
+            as: .image(size: view.intrinsicContentSize, traits: styleTrait),
```

Comment:
We needed to change this to use `intrinsicContentSize` or it would yield empty screenshots for BPKText. Other snapshots seem unharmed.